### PR TITLE
feat: add models for module timing

### PR DESCRIPTION
**Pull request by blaukc**
Added the module timing which has the day, start time and end time

### CODE REVIEW

File: src/main/java/seedu/address/model/module/Day.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import java.util.Locale;
+
+/**
+ * Represents a ModuleTiming's day in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDay(String)}
+ */
+public class Day {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Day should be in the format like Mon, Tue, Wed, Thu, Fri, Sat, Sun";
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("E");
```
Comment (yadunut) [inline src/main/java/seedu/address/model/module/Day.java:20]: would be good to add a comment on showing the format of a date in E pattern

**Review by taufiq — APPROVED** [review]
lgtmag

**Review by yadunut — APPROVED** [review]
LGTM!

**Review by AdityaB4 — APPROVED** [review]
LGTM!

**Pull request merged**